css/style.css

```
@@ -618,6 +647,27 @@ body {
     opacity: 1;
  }
  
+ /* — Swipe Header — */
+.swipe-header {
+   position: absolute;
+   top: -2rem;               /* tweak to sit right above your card */
+   left: 50%;
+   transform: translateX(-50%);
+   display: flex;
+   align-items: center;
+   gap: 0.5rem;
+ }
+ 
+ .instruction-text {
+   color: #b57f50;
+   font-size: 1.3rem;
+   white-space: nowrap;
+ }
+ 
+ .help-icon {
+   width: 28px;              /* matches your mockup */
+   height: auto;
+ }
  
  
```

```suggestion
 @media (max-width: 600px) {
  .swipe-hint {
    display: none;
  }
}
```

add this and you are good to go